index.js

```diff
@@ -392,31 +400,31 @@
   //. ```
   //.
   //. See also [`env`](#env).
-  S.create =
-  $.create({checkTypes: checkTypes, env: defaultEnv})('create',
-                                                      {},
-                                                      [Options, $.Object],
-                                                      createSanctuary);
+  function create(opts) {
+    var S = {
+      env: defaultEnv,
```

Comment:
This should be `opts.env` rather than `defaultEnv`, should it not? If others agree, I'll open a separate pull request for this change.

Reply:
#524